pkg/apis/crds/karpenter.k8s.aws_ec2nodeclasses.yaml

```diff
@@ -77,9 +77,21 @@ spec:
                       description: Tags is a map of key/value tags used to select
                         subnets Specifying '*' for a value selects all values for
                         a given tag key.
+                      maxProperties: 30
```

Comment:
Can we do this in reverse order, with this value being 20 and the number of terms being 30. I'd rather support more terms and less tag keys/values here